### PR TITLE
Restart telegraf service after hostname change

### PR DESCRIPTION
**Pull request by davidhemp**
Most of the login nodes are re-perused compute nodes which the hostname changed. These come up with the original compute node which is then changed after the image type installation. Telegraf therefore comes up with the compute hostname so we are restarted it after hostname change. These will restart the service on compute nodes as well but this shouldn't affect them.

### CODE REVIEW

**Review by dithwick — COMMENTED** [review]
Does this need a newline at the end?

File: final/60_telegraf_cleanup.sh [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# Restart telegraf to ensure a clean start after any hostname changes
+systemctl restart telegraf
```
Comment (davidhemp) [inline final/60_telegraf_cleanup.sh]:
```suggestion
systemctl restart telegraf

```

Comment (davidhemp) [reply final/60_telegraf_cleanup.sh]: Like this?

Comment (dithwick) [reply final/60_telegraf_cleanup.sh]: Yeah, I assume it's needed as git was making a point of it when I looked earlier.

**Review by dithwick — APPROVED** [review]

**Pull request merged**